ui-components/dialog.md

```diff
@@ -184,6 +183,66 @@ See the [dialog configuration options](#configurationoptions) documentation for
 }
 ```
 
+##### Dialog menu buttons
+
+A dialog menu button is a split button that can be used to provide a drop-down list of items in a dialog footer.
```

Comment:
Split button isn't the right term here, it should be a `dropdown button` or something instead, as split buttons are one that has two parts to it: one that is the primary and one that opens a sub menu.

Reply:
Agreed. This sentence also seems... awkward. I think mostly the usage of "provide" seems odd...